src/make.js

```diff
@@ -0,0 +1,17 @@
+module.exports = function make(...args) {
+  let concatArray = [...args];
```

Comment:
зачем? можно продолжить работать с args просто

Reply:
Это первое, что пришло в голову.

```suggestion
  let concatArray = [...args];
```
А лучше так?
```
function make(...args) {
  const getResult = (...newArgs) => {
    if (typeof newArgs[0] === 'function') {
      return args.reduce((sum));
    }
    args.push(...newArgs);
    return getResult;
  };

  return getResult;
}
```